src/foam/dao/FileJournal.js

```diff
@@ -327,14 +329,14 @@ foam.CLASS({
         JSONParser parser = getParser();
 
         try ( BufferedReader reader = getReader() ) {
-          for ( String entry ; ( entry = getEntry(reader) ) != null ; ) {
-            if ( SafetyUtil.isEmpty(entry)        ) continue;
-            if ( COMMENT.matcher(entry).matches() ) continue;
+          for ( String line ; ( line = getEntry(reader) ) != null ; ) {
```

Comment:
Why the revert back to line instead of entry?

Reply:
took it from a dated branch by accident; will fix